conf.yaml

```
@@ -1153,6 +1153,26 @@ contents:
                   -
                     repo:   apm-server
                     path:   CHANGELOG.asciidoc
+              - title:      APM Guide
+                prefix:     guide
+                index:      docs/integrations-index.asciidoc
+                current:    master
+                branches:   [ master ]
+                live:       master
```

`live` needs to be a list. That's what's causing `self-test` to fail.

```suggestion
                live:       [ master ]
```

---

Thanks!